src/stcal/ramp_fitting/ramp_fit.py

```diff
@@ -47,6 +47,10 @@ def create_ramp_fit_class(model, dqflags=None, suppress_one_group=False):
     ramp_data : ramp_fit_class.RampData
         The internal ramp class.
     """
+    if not suppress_one_group and hasattr(model, "zeroframe"):
```

Comment:
Does "hasattr()" actually work as needed here? Because zeroframe is defined in the RampModel, the attribute may always exist regardless of whether or not there's any data in the array. You might need to actually check for something like non-zero size of the zeroframe array.

Reply:
Before I new about the flag `model.meta.exposure.zero_frame` I used `hasattr`.  I thought I replaced all uses of `hasattr` with `model.meta.exposure.zero_frame`.